5.6/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM centos:centos7
+
+# MySQL image for OpenShift.
+#
+# Volumes:
+#  * /var/lib/mysql/data - Datastore for MySQL
+# Environment:
+#  * $MYSQL_USER - Database user name
+#  * $MYSQL_PASSWORD - User's password
+#  * $MYSQL_DATABASE - Name of the database to create
+#  * $MYSQL_ROOT_PASSWORD (Optional) - Password for the 'root' MySQL account
+
+MAINTAINER  Martin Nagy <mnagy@redhat.com>
+
+ENV MYSQL_VERSION=5.6 \
+    HOME=/var/lib/mysql
+
+LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database server" \
+      io.k8s.display-name="MySQL 5.6" \
+      io.openshift.expose-services="3306:mysql" \
+      io.openshift.tags="database,mysql,rh-mysql56"
```

Comment:
rh-mysql56 is name of the collection the image is based on, so let's use both, mysql56 and rh-mysql56 so the search works fine for both cases.